docs/sftt/sftt-api-spec-new.md

```diff
@@ -5,6 +5,36 @@ This is the official API specification for the SFTT back end. The back end is im
     In every request header is a JWT. The JWT will contain the user's id value and the user's privilege level.
     It is assumed that the user making the request (as specified in the JWT header) is the person the action is being performed by.
 
+## Enums
+
+### Team Role
+
+The team role enum is all the possible states a use can be in in relation to a team. This is stored in the users_teams table on the backend as the team_role field. The possible options are:
+
+- `None` - The user is not on the team. This is possible after a user is rejected from a team or leaves a team.
```

Comment:
We should probably continue the pattern of using all uppercase for our enums to be consistent

Reply:
I'll make sure to fix that in a PR on the backend